src/components/Range/index.tsx

```diff
@@ -0,0 +1,12 @@
+import Slider from 'react-input-slider';
+import { IRange } from './interfaces';
+import { mergeStyles } from './utils';
+
+const Range = ({ styles, ...props }: IRange) => <Slider styles={mergeStyles(props, styles)} {...props} />;
```

Comment:
Non-blocking because we're just moving this component. Do you think we should apply the defaultStyles by default?

Reply:
I think it should have at least a default value, maybe we can have variants in the future, but it's important to be able to use the component without setting any additional style configuration.

Do you have any suggestions for this one?

Reply:
I think there was a little confusion here! If the default styles are already being applied then that's great :)